### PR TITLE
Final Version Numbers

### PR DESCRIPTION
**Pull request by piotrpMSFT**
/cc @livarcocc, @barrytang, @srivatsn, @jgoshi, @krwq, @seancpeters

### CODE REVIEW

**Comment by jgoshi** [comment]
Did you forget to update the version number in $projectName$.csproj for the Mstest template? Maybe it's in there and I missed it...

File: TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj [hunk 1]
```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Sdk">
-      <Version>1.0.0-alpha-20161102-2</Version>
+      <Version>1.0.0-alpha-20161104-02</Version>
```
Comment (srivatsn) [inline TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj]: The versionis 20161104-2 (not 02)

Comment (piotrpMSFT) [inline TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj]: @srivatsn the original mail with SDK version number said `02`, so went with it. at first.

**Comment by piotrpMSFT** [comment]
@jgoshi great catch. That one had gotten out of sync with the rest of the product, probably merged after we last bumped sdk version. I'm looking for others. Note that the Perf test assets must stay locked back.

File: TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj (hunk 1 above)
Comment (srivatsn) [inline TestAssets/DesktopTestProjects/AppWithProjTool2Fx/App.csproj]: Ya sorry my bad. That's what I get for not copy pasting.

**Comment by ellismg** [comment]
@srivatsn Do you know what commit of dotnet/sdk this package corresponds to?

**Pull request merged**